shell/platform/android/io/flutter/embedding/engine/FlutterEngine.java

```diff
@@ -0,0 +1,39 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.embedding.engine;
+
+/**
+ * A single Flutter execution environment.
+ *
+ * WARNING: THIS CLASS IS EXPERIMENTAL. DO NOT SHIP A DEPENDENCY ON THIS CODE.
```

Comment:
Don't have a strong opinion on this, just wondering. Why not put this in android/io/flutter/experimental/embedding/engine first and move that folder to its parents once it's ready?

Reply:
I'm not aware of that being an existing process.  Also I don't think that will likely dissuade developers that really want to use this stuff but it will ensure that they're exceptionally broken when we cross the line to stable.  It's arguable whether that would be a bug or a feature.